Source/JavaScriptCore/dfg/DFGOperations.cpp

```diff
@@ -431,6 +452,27 @@ JSC_DEFINE_JIT_OPERATION(operationObjectToStringObjectSlow, JSString*, (JSGlobal
     return objectPrototypeToString(globalObject, thisValue);
 }
 
+JSC_DEFINE_JIT_OPERATION(operationReflectOwnKeys, JSArray*, (JSGlobalObject* globalObject, EncodedJSValue encodedObject))
+{
+    VM& vm = globalObject->vm();
+    CallFrame* callFrame = DECLARE_CALL_FRAME(vm);
+    JITOperationPrologueCallFrameTracer tracer(vm, callFrame);
+    auto scope = DECLARE_THROW_SCOPE(vm);
+
+    JSObject* object = JSValue::decode(encodedObject).toObject(globalObject);
+    RETURN_IF_EXCEPTION(scope, nullptr);
+    scope.release();
```

Comment:
Is it correct? I think Reflect.ownKeys does not accept non object input.

Reply:
Oh great catch, my bad copy & pasting, it indeed throws for non-objects. Fixed.